_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -1,54 +1,62 @@
 ---
 id: b8bb68
-name: Text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
+name: Text spacing in `style` attributes is not `!important`
 rule_type: atomic
 description: |
-  This rule checks that
+  This rule checks that the text spacing styles can be adjusted by user, i.e. that `!important` is not applied to the style, unless the style has already exceeded the specified metric for retaining content visibility and functionality.
 accessibility_requirements:
   wcag21:1.4.12: # Text Spacing (AA)
     forConformance: true
     failed: not satisfied
     passed: further testing needed
     inapplicable: further testing needed
-
 input_aspects:
   - DOM Tree
   - CSS Styling
-
 acknowledgments:
   authors:
     - Jey Nandakumar
 ---
 
 ## Applicability
 
-This rule applies to any HTML element that is [visible][] and has one of the following CSS properties set by the `style` attribute:
+This rule applies to any HTML element that is [visible][] and has a [declared][] value specified via the `style` attribute for any of the following CSS properties:
 
-- [word-spacing][]
-- [letter-spacing][]
-- [line-height][]
+- [word-spacing][],
+- [letter-spacing][],
+- [line-height][].
```

Comment:
i don't think we should use punctuations this way. It doesn't add anything. Why not use [conditional lists](https://github.com/act-rules/act-rules.github.io/blob/develop/pages/design/rule-design.md#listed-conditions)?